IntegrationTests/Framework/defs.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
Would be good to make sure the Python code follows a suitable standard Python style.

For rebench, I use black and PyLint:
 - https://github.com/smarr/ReBench/blob/master/.github/workflows/ci.yml#L41-L52
 - https://github.com/smarr/ReBench/blob/master/.pylintrc